redash/handlers/query_results.py

```diff
@@ -101,6 +151,28 @@ def options(self, query_id=None, query_result_id=None, filetype='json'):
         return make_response("", 200, headers)
 
     @require_permission('view_query')
+    def post(self, query_id=None, filetype='json'):
```

Comment:
This should return error 400 if the feature flag is disabled, as it's a security concern. Worth adding a comment here about this as well.